test/test_patch.rb

```diff
@@ -104,4 +104,16 @@ def test_merges_similar_ids_but_different_signs
       assert_equal(Zold::Amount.new(zld: 2.0), first.balance)
     end
   end
+
+  def test_not_allow_amount_exceed_max
+    coins = 2**63 - 1
+    FakeHome.new.run do |home|
+      wallet = home.create_wallet
+      time = Time.now
+      wallet.add(Zold::Txn.new(1, time, Zold::Amount.new(coins: 2), 'NOPREFIX', Zold::Id.new, '-'))
+      assert_raises RuntimeError do
```

Comment:
@liligga check if the expected error was raised. See `test_source.rb`  as an example.

Reply:
Can you help me with this a little bit. So, I realized I got this test wrong. Wanted to rewrite, but I can't because an error show as early as  `wallet.add big_amount` appears. So `patch.join` never gets hit. So I realized I have to create a wallet with transactions, but have no idea how.